Python/peephole.c

```diff
@@ -21,6 +21,7 @@
         (ABSOLUTE_JUMP(_Py_OPCODE(arr[i])) ? 0 : i+1))
 #define ISBASICBLOCK(blocks, start, end) \
     (blocks[start]==blocks[end])
+#define MAX_BYTECODE_OPT_ITERS 5
```

Comment:
This seems arbitrary, and quite low.
What the rationale for it being 5?

Reply:
This is the average change in the biggest files when compiling the CPython test suite and code and is a number lower enough so it does not take a time increment that is big enough to notice.

Technically you can leave it unbounded because the change will always make the bytecode smaller, but this will assure that is small enough to not be a difference.

Reply:
Can you add a brief comment to the code to this effect.

Reply:
Will do :)